src/main/java/com/synopsys/integration/detect/workflow/status/DetectStatusLogger.java

```diff
@@ -157,6 +164,20 @@ private void logDetectOperations(IntLogger logger, List<Operation> operations) {
         logger.debug("");
     }
 
+    private void logAutonomousDecisions(IntLogger logger, AutonomousManager autonomousManager) {
+        if (autonomousManager.getAutonomousScanEnabled()) {
+            logger.debug("====== Autonomous Scan ======");
```

Comment:
Should be INFO level logs.

Reply:
This is debug because as per IDETECT-4290, these decisions and the file location should only appear in logs in diagnostic mode.

Reply:
Are there any logs that would indicate at a high level key decisions made by autonomous mode at INFO level?

Reply:
Not as of now. We can add an INFO log mentioning we are in autonomous mode. File location and scan decisions can be displayed under the hood (in debug mode) as the user need not look at those (i.e. they are for system use only)

Reply:
User needs to get feedback on what autonomous mode did for **key decisions** should be at INFO level. Otherwise, the feature will not be user friendly. I am not referring to setting file content but at a high level - scan type(s) and their scan modes at least should be printed at INFO level.

Reply:
Sure we can discuss where to show these details. This can be shown during the workflow as well amongst other INFO level log statements while the decisions are being taken. I see some info loggers already existing in ScanTypeDecider - can the loggers be added there? We can leave those tweaks outside of the scope of this PR.